lib/modules/manager/gomod/artifacts.ts

```diff
@@ -296,13 +296,18 @@ export async function updateArtifacts({
       logger.debug('go mod tidy command skipped');
     }
 
-    const tidyOpts = config.postUpdateOptions?.includes('gomodTidy1.17')
-      ? ' -compat=1.17'
-      : '';
+    let tidyOpts = '';
+    if (config.postUpdateOptions?.includes('gomodTidy1.17')) {
+      tidyOpts = ' -compat=1.17';
+    } else if (config.postUpdateOptions?.includes('gomodTidyE')) {
```

Comment:
shouldn't we allow both options?

Reply:
I can change that if you like. So if both options are present it adds both arguments. Will implement that next week. 👍

Reply:
yes, please do .  but only if they are allowed to be used together otherwise warn

Reply:
Done